src/components/Checkbox/index.tsx

```diff
@@ -1,19 +1,27 @@
-import React, { Component } from 'react';
-import PropTypes from 'prop-types';
+import * as React from 'react';
```

Comment:
For consistency, do:

```suggestion
import React from 'react';
```